src/components/AuthForm/authForm.test.jsx

```diff
@@ -0,0 +1,136 @@
+import { describe, expect, it, vi } from "vitest";
+import { render, waitFor, screen } from "@testing-library/react";
+import AuthForm from ".";
+import { Provider } from "react-redux";
+import { configureStore } from "@reduxjs/toolkit";
+import authSlice from "features/auth/authSlice";
+import { MemoryRouter } from "react-router-dom";
+import userEvent from "@testing-library/user-event";
+import registerSlice from "features/register/registerSlice";
```

Comment:
Can we rearrange the imports? One best practice we follow while importing files is that we follow a order of: React imports > Other library Imports > Internal file imports > css imports

```suggestion
import { describe, expect, it, vi } from "vitest";
import { render, waitFor, screen } from "@testing-library/react";
import { Provider } from "react-redux";
import { configureStore } from "@reduxjs/toolkit";
import authSlice from "features/auth/authSlice";
import { MemoryRouter } from "react-router-dom";
import userEvent from "@testing-library/user-event";
import registerSlice from "features/register/registerSlice";
import AuthForm from ".";
```

Reply:
Resolved